pages/1.9/release-notes/1.9.9/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.9.9
+title: Release Notes for 1.9.9
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.9.9
+---
+
+DC/OS 1.9.9 was released on July 10, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.9/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.9.9 includes the following:
+
+- Updated to Mesos 1.2.x ([change log](https://github.com/dcos/dcos/blob/1.9.9/packages/mesos/buildinfo.json)).
```

Comment:
If this is development version we need to add -dev

Reply:
CC @tillt

Reply:
@tillt is this entry now at least not wrong?

Reply:
This is mesos version 1.2.3. It could be verified from here - https://github.com/mesosphere/mesos/blob/dcos-mesos-mesosphere/1.2.x-nightly-cb79076/CHANGELOG